modules/kubernetes-addons/istio/main.tf

```diff
@@ -0,0 +1,138 @@
+resource "kubernetes_namespace" "istio_system" {
+  metadata {
```

Comment:
namespace creation and name used should be configurable - see other examples for reference

Reply:
Istio control plane can be installed only in `istio-system` namespace